src/version.h

```diff
@@ -37,4 +37,7 @@ static const int NO_BLOOM_VERSION = 170004;
 //! - MSG_WTX type defined, which contains two 32-byte hashes.
 static const int CINV_WTX_VERSION = 170014;
 
+//! disconnect from testnet peers older than this proto version
+static const int MIN_TESTNET_PEER_PROTO_VERSION = 170040;
```

Comment:
I confirmed that this is the protocol version currently used on `nu5-consensus` and that 4.7.0 will use.

Reply:
Wait, I thought 4.7.0 would use 170050?

Reply:
I don't know what I was thinking when half-hour-past me wrote that comment. 4.7.0 will indeed use 170050. Current `nu5-consensus` is using 170040, and _that_ is what this PR needs to use (so that nodes built from it immediately benefit).